src/test/java/edu/neu/coe/info6205/randomwalk/RandomWalkTest.java

```diff
@@ -86,4 +89,16 @@ public void testRandomWalk2() {
         for (int i = 0; i < 5000; i++)
             assertNotSame(0, RandomWalk.randomWalkMulti(1, 1));
     }
-}
\ No newline at end of file
+
+    @Test
+    public void testRandomWalk3() {
+        Random random = new Random();
+        for (int i = 0; i < 100; i++) {
+            int steps = random.nextInt(200);
+            double expected = Math.sqrt(steps);
+            double average = RandomWalk.randomWalkMulti(steps, 10000);
+            System.out.printf("steps: %d Expected value: %.4f Actual value: %.4f Difference: %.4f\n", steps, expected, average, expected-average);
+            assertEquals(expected, average, 4);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.